tensorflow/core/api_def/base_api/api_def_Bitcast.pbtxt

```diff
@@ -12,6 +12,33 @@ If `T` is smaller than `type`, the operator requires that the rightmost
 dimension be equal to sizeof(`type`)/sizeof(`T`). The shape then goes from
 [..., sizeof(`type`)/sizeof(`T`)] to [...].
 
+tf.bitcast() is used when you have to cast to an unsigned dtype (e.g. tf.uint8)
+where tf.cast() will not work effectively.
```

Comment:
What do you mean by "effectively" here? In the examples below you show int8 -> uint8 but `tf.cast` vs. `tf.bitcast` would give the same result.

Perhaps give an example of where they differ (e.g. when casting to dtypes of a different number of bits, or float->int):

```python
>>> tf.cast(1.234, tf.int32)
<tf.Tensor: ... shape=(), dtype=int32, numpy=1>

>>> tf.bitcast(1.234, tf.int32)
<tf.Tensor: ... shape=(), dtype=int32, numpy=1067316150>
```

Reply:
ok i will remove that, but can I add an example where tf.complex128 gives error in tf.bitcast while work with tf.cast where it makes imaginary part 0.